openlabcmd/openlabcmd/plugins/nodepool/orphanresource.py

```diff
@@ -0,0 +1,130 @@
+import datetime
+import iso8601
+import subprocess
+
+import openstack
+
+from openlabcmd.plugins.base import Plugin
+
+# DAYs
+RESOURCE_TIMEOUT = 1
+
+
+class OrphanResourcePlugin(Plugin):
+    ptype = 'nodepool'
+    name = 'orphan_resource'
+
+    # This Plugin is only for checking, as it's a dangerous action for each.
+    def __init__(self, cloud, config):
+        super(OrphanResourcePlugin, self).__init__(cloud, config)
+        self.openstack_client = openstack.connect(cloud)
```

Comment:
In future we can move the openstack_client into a separate class, and the plugin is using the openstacksdk will be defined as:
```
class OrphanResourcePlugin(Plugin, OpenstackSDKHelper):
```

Reply:
I mean if we have lots of other methods use sdk to do some complex thing, but now we can keep it.

Reply:
Good point, it can be done once more plugin using openstacksdk